### DISCUSSION TITLE
Remove Clutter in the Top Right

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2157] [Ideas]
<img width="1065" alt="Screen Shot 2021-12-07 at 11 42 06 AM" src="https://user-images.githubusercontent.com/83455454/145079679-0d812f93-5d0b-4d82-a853-47039227735b.png">

This isn't top priority by any means but I definitely think it would be a very good improvement to remove the clutter in the top right. Thoughts?

**Comment by Miodec** [comment]
I thought about it but any changes would result in hiding options behind menus. Im open to suggestions but i couldnt figure out anything i liked.
